### PR TITLE
Rename the function name

### PR DESCRIPTION
**Pull request by chansuke**
Rename the function names for readability and [consistency](https://github.com/kubernetes/test-infra/blob/master/prow/github/client.go#L126-L146).

### CODE REVIEW

**Review by petr-muller — COMMENTED** [review]
I think there are places that import this code but we did not commit to a rigid API stability and this is fairly easy to adjust to, so I'm good with it.

/test all

File: prow/github/client.go [hunk 1]
```diff
@@ -135,8 +135,8 @@ type PullRequestClient interface {
 	ListPullRequestComments(org, repo string, number int) ([]ReviewComment, error)
 	CreatePullRequestReviewComment(org, repo string, number int, rc ReviewComment) error
 	ListReviews(org, repo string, number int) ([]Review, error)
-	ClosePR(org, repo string, number int) error
-	ReopenPR(org, repo string, number int) error
+	ClosePullRequest(org, repo string, number int) error
+	ReopenPullRequest(org, repo string, number int) error
```
[the remark this answers is not in the document]
Comment (chansuke) [reply prow/github/client.go:139]: @petr-muller Thank you so much for your comment. I've fixed the `ListPRCommits`.

**Review by petr-muller — APPROVED** [review]

**Review by petr-muller — APPROVED** [review]

**Pull request merged**